examples/absolute-scale/js/button-end-ar-session.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * ButtonEndARSession
+ * Shows a simple HTML buttton when ARSession has started.
+ * When clicked - kill any running AR session.
+ */
+import {Component} from '@wonderlandengine/api';
+import {ARSession} from '@wonderlandengine/ar-tracking';
+
+export class ButtonEndARSession extends Component {
+    static TypeName = 'button-end-ar-session';
+
+    xrEndButton!: HTMLButtonElement;
```

Comment:
I wonder if you could maybe create a package `ar-tracking-example-utils` that is shared by all? Or put it in common-components? Kind of got used to that folder now :)

Reply:
Yeah, that's a temporary workaround for at least make the examples run.
nightly-20230508 would crash on startup with commont-components. (files outside the root of the project)